cmd/collector/main.go

```diff
@@ -95,3 +107,57 @@ func logOptions(loggingConfigPath *string) ([]zap.Option, error) {
 
 	return l.Options()
 }
+
+func checkManagerConfig(configPath *string) error {
+	_, statErr := os.Stat(*configPath)
+
+	switch {
+	case statErr == nil:
+		// We found the file just return nil
+		return nil
+	case errors.Is(statErr, os.ErrNotExist):
+		// Endpoint is only required env
+		if ep, ok := os.LookupEnv(endpoint); ok {
```

Comment:
This is super nipicky; But would it be better to do something like:
```go
ep, ok := os.LookupEnv(endpoint)
if !ok {
    return statErr
}
// stuff that was previously nested in the if statment
...
return nil
```

To avoid this nesting here?

Reply:
Yeah I would agree with that. The very first `os.LookupEnv` for `endpointENV` should probably be what @BinaryFissionGames suggested. This is more idiomatic Go with avoiding nesting by returning early.